dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/converters/RorConverter.java

```diff
@@ -0,0 +1,82 @@
+package edu.harvard.iq.dataverse.api.converters;
+
+import edu.harvard.iq.dataverse.api.dto.RorEntryDTO;
+import edu.harvard.iq.dataverse.persistence.ror.RorData;
+import edu.harvard.iq.dataverse.persistence.ror.RorLabel;
+import edu.harvard.iq.dataverse.search.ror.RorDto;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.ejb.Stateless;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * Simple converter for Ror objects.
+ */
+@Stateless
+public class RorConverter {
+
+    // -------------------- LOGIC --------------------
+
+    public RorData toEntity(RorEntryDTO entry) {
+        RorData converted = new RorData();
+
+        converted.setRorId(extractRor(entry.getId()));
+        converted.setName(entry.getName());
+
+        if (entry.getCities().length > 0) {
+            converted.setCity(entry.getCities()[0].getCity());
+        }
+
+        if (entry.getLinks().length > 0) {
+            converted.setWebsite(entry.getLinks()[0]);
+        }
+
+        if (entry.getCountry() != null) {
+            converted.setCountryName(entry.getCountry().getCountryName());
+            converted.setCountryCode(entry.getCountry().getCountryCode());
+        }
+
+        converted.getAcronyms().addAll(Arrays.asList(entry.getAcronyms()));
+        converted.getNameAliases().addAll(Arrays.asList(entry.getAliases()));
+        converted.getLabels().addAll(
+                Arrays.stream(entry.getLabels())
+                      .map(l -> new RorLabel(l.getLabel(), l.getIso639()))
+                      .collect(Collectors.toSet()));
+
+        return converted;
+    }
+
+    public RorDto toSolrDto(RorData entry) {
```

Comment:
I don't think it's the right class for this method. IMO it should be closer to ror search package. (RorConverter seems to be for api purpouse only)

Reply:
Then I think moving this class to not api package should be sufficient, I think this kind of class is so simple that it can be used for all ROR based data.